src/cmds/pbsdsh.c

```diff
@@ -239,7 +350,9 @@ main(int argc, char *argv[], char *envp[])
 				err = 1;
 				break;
 		}
+		if (targethost) break;
```

Comment:
Please put the break on its own line.

Also, are you sure you want to break?  You'll stop parsing arguments if targethost is true.  Any arguments after -h won't be parsed.

Reply:
The thought at first was we should stop parsing args once you met a -h entry, as you could not mix -h -n -c. 
We can do better here ofc, will try to do some improvement here.